src/NuGetPackager/PackageCreator.cs

```diff
@@ -9,7 +9,7 @@
 
     class PackageCreator : IPropertyProvider
     {
-        readonly Dictionary<string, Func<string>> propertyAssignments; 
+        readonly Dictionary<string, Func<string>> propertyAssignments;
```

Comment:
Fertile ground for @SimonCropp 's "readonly" inquisition :)

![the_spanish_inquisition_zpsd96e40e9](https://cloud.githubusercontent.com/assets/125028/14016906/7e23dfbe-f1c3-11e5-9cff-41e46d41b856.gif)